stacks/spica/src/bucket/bucket.module.ts

```diff
@@ -53,6 +53,9 @@ import {BucketInitializer} from "./services/bucket.initializer";
 import * as fromBucket from "./services/bucket.reducer";
 import {BucketService} from "./services/bucket.service";
 import {CommonModule as SpicaCommon} from "@spica-client/common";
+import {MatDatepickerModule} from "@angular/material/datepicker";
```

Comment:
Do we really need these two modules. (MatDatepickerModule, MccTimerPickerModule)